src/features/commands.ts

```diff
@@ -410,6 +412,30 @@ To integrate it into your editor: https://prettier.io/docs/en/editors.html`,
         }
       });
     }
+  },
+  {
+    words: ["@here", "@everyone"],
+    help: "",
+    handleMessage: msg => {
+      if (!msg || !msg.guild) {
+        return;
+      }
+
+      const member = msg.guild.member(msg.author.id);
+
+      if (!member || isStaff(member)) {
+        return;
+      }
+
+      msg.channel.send({
+        embed: {
+          title: "Tsk tsk.",
+          type: "rich",
+          description: `Please do **not** try to use \`@here\` or \`@everyone\` - there are ${msg.guild.memberCount} members in Reactiflux. Everybody here is a volunteer, and somebody will help you if they can.`,
```

Comment:
Although some people also try to ping everyone when they've made something and they're trying to share it in `#i-made-this`, maybe we could also add a short blurb for that scenario?

Reply:
`Somebody will respond if they can`?

Reply:
works for me